envs/example/defaults.yml

```diff
@@ -122,6 +122,10 @@ ceph:
   monitor_group_name: ceph_monitors
   osd_group_name: ceph_osds
   public_and_cluster_network:
+  #devices:
+  #  - /dev/sdb
+  #  - /dev/sdc
+  #  - /dev/sdd
 
```

Comment:
Swift has "disks" param and excludes "/dev" from it....We should try to keep it consistent since the plan is to use common playbook to "ready" nodes